src/stdatamodels/jwst/transforms/models.py

```diff
@@ -106,69 +162,62 @@ class GrismObject(namedtuple('GrismObject', ("sid",
     GrismObject(order_bounding={"+1":((xmin,xmax),(ymin,ymax)),"+2":((2,3),(2,3))})
 
     """
+
     __slots__ = ()  # prevent instance dictionary for lower memory
 
-    def __new__(cls,
-                sid=None,
-                order_bounding={},
-                sky_centroid=None,
-                partial_order={},
-                waverange=None,
-                sky_bbox_ll=None,
-                sky_bbox_lr=None,
-                sky_bbox_ur=None,
-                sky_bbox_ul=None,
-                xcentroid=None,
-                ycentroid=None,
-                is_extended=None,
-                isophotal_abmag=None):
-
-        return super(GrismObject, cls).__new__(cls,
-                                               sid=sid,
-                                               order_bounding=order_bounding,
-                                               sky_centroid=sky_centroid,
-                                               partial_order=partial_order,
-                                               waverange=waverange,
-                                               sky_bbox_ll=sky_bbox_ll,
-                                               sky_bbox_lr=sky_bbox_lr,
-                                               sky_bbox_ur=sky_bbox_ur,
-                                               sky_bbox_ul=sky_bbox_ul,
-                                               xcentroid=xcentroid,
-                                               ycentroid=ycentroid,
-                                               is_extended=is_extended,
-                                               isophotal_abmag=isophotal_abmag)
+    def __new__(
+        cls,
+        sid=None,
+        order_bounding={},  # noqa: B006
```

Comment:
looks to me like these could be easily fixed by setting the default to `None` as suggested by the rule

Reply:
Maybe but that's a change in behavior. Using a modifiable argument allows things like caching values. It's unclear to me at a cursory glance why this is `{}` and others are `None`. Without digging further into it I'm not sure if that's what's happening here. This line is also uncovered. I'd rather go with the safer bet of adding a `noqa` than changing the behavior.

Reply:
Ok. Happy to include this in fixes associated with https://github.com/spacetelescope/stdatamodels/issues/388